perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/scheduling/TaskScheduler.java

```diff
@@ -371,17 +371,19 @@ protected TaskScheduled sendToEngine(Task task) {
 				// when service is blocked globally in Perun or on facility as a whole.
 				return DENIED;
 			}
+		} else {
+			log.error("No destination found for given service: {}", service.getId());
```

Comment:
From the perspective of logs I would consider this to be warning or just debug. Also destinations are bound to Task, since its combination of facility/service, where the data is provisioned.
Also see generic template for log messages in Dispatcher/Engine, eg. `[{}] Removed blocked destination: {}`. They are like this in order to easily "grep" the whole Task processing.